static/styles/settings.css

```diff
@@ -526,7 +526,7 @@ input[type=checkbox].inline-block {
 }
 
 .add-new-filter-box button {
-    margin-left: calc(10em + 20px) !important;
+    margin-left: calc(10em - -20px) !important;
```

Comment:
I think this one should be fixed. GIF attached
![calc-4](https://user-images.githubusercontent.com/17237412/36866942-bf70b752-1db9-11e8-951a-4b6a84bc537f.gif)